Auto-Correct Private DNS Names Networking Issue (#1926)
ASFHyP3/hyp3: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

> ## [4.4.0]

Reviewer: ```suggestion
## [4.3.2]
```

> ### Added
> - Lambda function that disables `Enable Private DNS Names` for VPC endpoint.

Reviewer: ```suggestion
### Fixed
- Added a Lambda function that disables `Enable Private DNS Names` for VPC endpoint.
```

> - Added a Lambda function that disables `Enable Private DNS Names` for VPC endpoint.

Reviewer: ```suggestion
- Added a Lambda function that sets `Private DNS names enabled` to false for VPC endpoint.
```

> - Lambda function that checks to ensure that `Enable Private DNS Names` is enabled for endpoints

Reviewer: ```suggestion
- Lambda function that disables `Enable Private DNS Names` for VPC endpoint.
```
